datalad_dataverse/remote.py

```diff
@@ -34,16 +35,26 @@ def initremote(self):
             raise RuntimeError("Cannot find dataset")
 
     def prepare(self):
-        raise
         pass
 
     def checkpresent(self, key):
-        raise
-        pass
+        api = NativeApi(self.annex.getconfig('url'), os.environ.get('DATAVERSE_API_TOKEN', None))
+        dataset = api.get_dataset(identifier=self.annex.getconfig('doi'))
+
```

Comment:
I think a `dataset.raise_for_status()` would be good here, too. Just as you did below in `transfer_store`.